lib/webview_windows.dart

```diff
@@ -313,6 +313,14 @@ class WebviewController extends ValueNotifier<WebviewValue> {
     return _methodChannel.invokeMethod('reload');
   }
 
+  /// Stops all navigations and pending resource fetches.
+  Future<bool?> stop() async {

```

Comment:
Why does this return a nullable bool? Shouldn't it just return a `Future<void>`?

Btw., now that you've changed all those methods to return a future,  the `async` keyword can be removed.

Reply:
Initially I had it as a void but since it returns a result I thought I should set it as bool, it might fail to stop?

Check the update now

Reply:
Ok, as far as I can see, you're not considering the return value of `webview_->Stop()` in `webview_bridge.cc` anyway, so the only reason why this might fail is an issue with the message channel itself.

But I need to admit that I was not really consistent. Some methods like `postWebMessage` check if the actual WebView invocation was successful, while others like `reload` don't. So let's always check the return values for future method implementations in `webview_bridge.cc`.

Regarding the `async` keyword on the dart side. Please forget what I just said. I didn't see the line that checks if the `WebviewController` was already disposed while reviewing this, so let's just keep those methods async ;)